### DISCUSSION TITLE
Enter key in the num pad should be handled the same as the main enter key

### DISCUSSION THREAD

**Discussion by anderson-arlen** [zed-industries/zed#12757] [Feature Requests]
Something that has been bugging the hell out of me is that every time I hit the enter key on my num pad, nothing happens. Is there a way to bind this key to the same action as the main enter key?

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
